lib/Yam/ControlCenter.pm

```diff
@@ -0,0 +1,374 @@
+}
+
+
+sub _select_system {
+    my ($self) = @_;
+    $self->{slb_groups}->select('System');
+    workaround_bsc1206966();
+}
+
+sub _select_boot_loader {
+    my ($self) = @_;
+    $self->{slb_progs}->select('Boot Loader');
+}
+
+sub _select_date_and_time {
+    my ($self) = @_;
+    $self->{slb_progs}->select('Date and Time');
+}
+
+sub _select_kernel_kdump {
+    my ($self) = @_;
+    $self->{slb_progs}->select('Kernel Kdump');
+}
+
+sub _select_language {
+    my ($self) = @_;
+    $self->{slb_progs}->select('Language');
+}
+
+sub _select_network_settings {
+    my ($self) = @_;
+    $self->{slb_progs}->select('Network Settings');
+}
+
+sub _select_services_manager {
+    my ($self) = @_;
+    $self->{slb_progs}->select('Services Manager');
+}
+
+
+sub _select_hardware {
+    my ($self) = @_;
+    $self->{slb_groups}->select('Hardware');
+    workaround_bsc1206966();
+}
+
+sub _select_printer {
+    my ($self) = @_;
+    $self->{slb_progs}->select('Printer');
+}
+
+sub _select_system_keyboard_layout {
+    my ($self) = @_;
+    $self->{slb_progs}->select('System Keyboard Layout');
+}
+
+
+sub _select_network_services {
+    my ($self) = @_;
+    $self->{slb_groups}->select('Network Services');
+    workaround_bsc1206966();
+}
+
+sub _select_hostnames {
+    my ($self) = @_;
+    $self->{slb_progs}->select('Hostnames');
+}
+
+sub _select_ntp_configuration {
+    my ($self) = @_;
+    $self->{slb_progs}->select('NTP Configuration');
+}
+
+sub _select_proxy {
+    my ($self) = @_;
+    $self->{slb_progs}->select('Proxy');
+}
+
+sub _select_remote_administration {
+    my ($self) = @_;
+    $self->{slb_progs}->select('Remote Administration (VNC)');
+}
+
+sub _select_iscsi_initiator {
+    my ($self) = @_;
+    $self->{slb_progs}->select('iSCSI Initiator');
+}
+
+
+sub _select_security_and_users {
+    my ($self) = @_;
+    $self->{slb_groups}->select('Security and Users');
+    workaround_bsc1206966();
+}
+
+sub _select_firewall {
+    my ($self) = @_;
+    $self->{slb_progs}->select('Firewall');
+}
+
+sub _select_user_and_group_management {
+    my ($self) = @_;
+    $self->{slb_progs}->select('User and Group Management');
+}
+
+
+sub _select_support {
+    my ($self) = @_;
+    $self->{slb_groups}->select('Support');
+    workaround_bsc1206966();
+}
+
+sub _select_release_notes {
+    my ($self) = @_;
+    $self->{slb_progs}->select('Release Notes');
+}
+
+
+sub _select_miscellaneous {
+    my ($self) = @_;
+    $self->{slb_groups}->select('Miscellaneous');
+    workaround_bsc1206966();
+}
+
+sub _select_display_system_log {
+    my ($self) = @_;
+    $self->{slb_progs}->select("Display the system's log (/var/log/messages)");
+}
+
+sub _select_systemd_journal {
+    my ($self) = @_;
+    $self->{slb_progs}->select('Systemd Journal');
+}
+
+sub _select_vendor_driver {
+    my ($self) = @_;
+    $self->{slb_progs}->select('Vendor Driver');
+}
+
+
+sub _run {
+    my ($self) = @_;
+    $self->{btn_run}->click();
+}
+
+# methods
```

Comment:
they are all methods, it is a way to express that some are public and some are not, but this is very from other programming languages, so I would not bother to put a comment, or if you think is helpful,
`public methods`, `private methods` for example.